curriculum/challenges/english/25-front-end-development/lab-url-shortener-microservice/bd7158d8c443edefaeb5bd0e.md

```diff
@@ -0,0 +1,107 @@
+---
+id: bd7158d8c443edefaeb5bd0e
+title: URL Shortener Microservice
+challengeType: 4
```

Comment:
I noticed the forumTopicId was removed here. Do we not want that anymore? I feel like we should keep it.